tools/ci_build/github/linux/docker/Dockerfile.manylinux2010_gpu

```diff
@@ -0,0 +1,66 @@
+FROM quay.io/pypa/manylinux2010_x86_64:latest
```

Comment:
I guess this requires nvidia-docker runtime v2, right?

Reply:
The manylinux2010 docker images has no adherence whatsoever to NVIDIA docker.
The relevant bits are line 54 of that file.
It probably will not work with nvidia-docker v1, if this is an issue, I can add the lines needed for that.